linchpin/hooks/core_action_managers.py

```diff
@@ -0,0 +1,13 @@
+from subprocess_action_manager import SubprocessActionManager
```

Comment:
This should be put in the __init__.py instead as it's just setting up data for the package and modules inside.